util-cache/src/main/scala/sbt/internal/util/codec/HashedVirtualRefFormats.scala

```diff
@@ -0,0 +1,29 @@
+package sbt
+package internal
+package util
+package codec
+
+import sjsonnew.{ BasicJsonProtocol, IsoString }
+import xsbti.HashedVirtualFileRef
+
+trait HashedVirtualFileRefFormats { self: BasicJsonProtocol =>
+
+  /**
+   * A string representation of HashedVirtualFileRef, delimited by `>`.
+   */
+  def hashedVirtualFileRefToStr(ref: HashedVirtualFileRef): String =
+    s"${ref.id}>${ref.contentHashStr}/${ref.sizeBytes}"
```

Comment:
Do they use the same formatting in Bazel?

Reply:
Nope. As far as I know, there isn't a string representation for this. The closest concept is `OutputFile` - https://github.com/bazelbuild/remote-apis/blob/96942a2107c702ed3ca4a664f7eeb7c85ba8dc77/build/bazel/remote/execution/v2/remote_execution.proto#L1229-L1254